connected peers (#64)
AAU-Dat/P9-ZKP: report/src/sections/08-future-works.tex

This solution would make it harder for the attacker to de-anonymize validators and perform the~\gls{dos} attack.
Though, this would also require new validators in the system to have a set of trusted peers that they can use as relays for their messages.
This would also increase the threshold for entry in the system for validators as well as increase the latency within the system due to the increased number of hops the messages would have to go through.

\subsection{Further research}\label{subsec:further-research}

> Due to time constraints, we were able to fully reproduce the attack mentioned in~\cite{heimbach2024deanonymizingethereumvalidatorsp2p}.

Reviewer: ```suggestion
Due to time constraints, we were not able to fully reproduce the attack mentioned in~\cite{heimbach2024deanonymizingethereumvalidatorsp2p}.
```

> In the original paper, they run treir experiment on the mainnet, while we ran our experiment on a testnet.

Reviewer: ```suggestion
In the original paper, they run their experiment on the mainnet, while we ran our experiment on a testnet.
```

This means that the results of the two experiments could differ due to the differences in the two networks.
With more time, it would be interesting to run our experiment on the mainnet to see if the results would more closely align with the original paper.

Another place to look for further research would be to implement the~\gls{dos} attack as a follow-up to this paper.

> Since this paper mainly focuses on the data collection part of the attack, it would be interesting to see how the~\gls{dos} attack would use the data collected to stop a proposer from proposing a block.

Reviewer: ```suggestion
Since this paper mainly focuses on the data collection part of the attack, it would be interesting to see the consequences of the~\gls{dos} attack trying to stop a proposer from proposing a block using the data collected.
```
